packages/api/src/router/widgets/stocks.ts

```diff
@@ -0,0 +1,23 @@
+import { z } from "zod";
+
+import { fetchStockPriceHandler } from "@homarr/request-handler/stock-price";
+
+import { stockPriceTimeFrames } from "../../../../widgets/src/stocks";
+import { createTRPCRouter, publicProcedure } from "../../trpc";
+
+const stockPriceInputSchema = z.object({
+  stock: z.string().nonempty(),
+  timeRange: z.enum([...stockPriceTimeFrames.range]),
```

Comment:
Does it not work with `z.enum(stockPriceTimeFrames.range)`?
If it works, fix interval as well